INSTALL.md

```diff
@@ -539,13 +539,16 @@ email will be disabled.
 
 The easiest way to create a new user is to do so from a client like [Element](https://element.io/).
 
-Alternatively you can do so from the command line if you have installed via pip.
+Alternatively, you can do so from the command line if you have installed synapse via package.
 
 This can be done as follows:
 
 ```sh
+$ # These steps are needed if installed via pip.
```

Comment:
```suggestion
$ # These steps are needed if you installed via pip.
```

Reply:
Hey, I think it's better for docs to be imperative or passive, hope you're satisfied with the update